hapi/hedera-protobufs/services/consensus_topic_info.proto

```diff
@@ -87,4 +88,19 @@ message ConsensusTopicInfo {
      * The ledger ID the response was returned from; please see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-198.md">HIP-198</a> for the network-specific IDs. 
      */
     bytes ledger_id = 9;
+
+    /**
+    * Access control for update/delete of custom fees. Null if there is no key.
+    */
+    Key fee_schedule_key = 10;
+
+    /**
+      * If the transaction contains a signer from this list, no custom fees are applied.
+      */
+    repeated Key free_messages_key_list = 11;
```

Comment:
Rather than `repeated Key`, it _might_ make more sense to make this a `KeyList` (not sure the reason this was done as a repeated instead).